documentation/CHANGELOG_EXAMPLE.md

```diff
@@ -228,33 +194,115 @@ Changes compared to r2.3
   - Especially a lot of the parameter names changed in line with the agreed glossary within CAMARA Commonalities
 - This is an alpha version, it should be considered as a draft.
 - There are bug fixes to be expected and incompatible changes in upcoming versions. 
-- The release is suitable for implementors, but it is not recommended to use the API with customers in productive environments.
+- The version is suitable for implementors, but it is not recommended to use the API with customers in productive environments.
+
+- API definition **with inline documentation**:
+  - [View it on ReDoc](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/camaraproject/QualityOnDemand/r1.2/code/API_definitions/qod-api.yaml&nocors)
+  - [View it on Swagger Editor](https://editor.swagger.io/?url=https://raw.githubusercontent.com/camaraproject/QualityOnDemand/r1.2/code/API_definitions/qod-api.yaml)
+  - OpenAPI [YAML spec file](https://github.com/camaraproject/QualityOnDemand/blob/r1.2/code/API_definitions/qod-api.yaml)
+
+### Added
+
+* Added basic tests with Cucumber framework using Java and Maven implementation by @mdomale in https://github.com/camaraproject/QualityOnDemand/pull/134
+
+### Changed
+
+* Aligned error format with CAMARA design guidelines by @jlurien in https://github.com/camaraproject/QualityOnDemand/pull/104
+* Renamed properties to new terms agreed in CAMARA Commonalitites by @jlurien in https://github.com/camaraproject/QualityOnDemand/pull/129
+* CAMARA documentation is now embedded within the OAS definition, and not separate by @jlurien in https://github.com/camaraproject/QualityOnDemand/pull/151
+
+### Fixed
+
+* Added error code 501 "Not Implemented" by @dfischer-tech in https://github.com/camaraproject/QualityOnDemand/pull/124
+  
+### Removed
+
+* Removed format lines from Datatypes `Ipv4Address` and `Ipv6Address` by @tlohmar in https://github.com/camaraproject/QualityOnDemand/pull/177
+* Removed markdown documentation (now embedded within the OAS definition, see above)
+
+## New Contributors
+* @jlurien made their first contribution in https://github.com/camaraproject/QualityOnDemand/pull/104
+* @dfischer-tech made their first contribution in https://github.com/camaraproject/QualityOnDemand/pull/124
+* @mdomale made their first contribution in https://github.com/camaraproject/QualityOnDemand/pull/134
+
+**Full Changelog**: https://github.com/camaraproject/QualityOnDemand/compare/r1.1...r1.2
+
+# r1.1 - alpha
+
+## Release Notes
+
+This release contains the definition and documentation of
+* quality-on-demand v0.10.0-alpha.1
+
+## quality-on-demand v0.10.0-alpha.1
+
+- **This version contains significant changes compared to v0.9.0, and it is not backward compatible**
+  - Especially a lot of the parameter names changed in line with the agreed glossary within CAMARA Commonalities
+- **This is only the pre-release, it should be considered as a draft of the upcoming release v0.9.0**
```

Comment:
```suggestion
- **This is only the pre-release, it should be considered as a draft of the upcoming release v0.10.0**
```
The root cause here is that this two alpha versions are derived from the v0.9.0 content and not from the v0.10.0-rc.1 content as they should be.